app/uk/gov/hmrc/teamsandrepositories/services/GithubV3RepositoryDataSource.scala

```diff
@@ -102,31 +104,31 @@ class GithubV3RepositoryDataSource(
     }
   }
 
-
   def getRepositoryDetailsFromGithub(repository: GhRepository): Future[GitRepository] =
     for {
-      manifest                <- withCounter(s"github.open.fileContent") {
-                                   gh.getFileContent("repository.yaml", repository.name, HMRC_ORG)
-                                 }
-      maybeManifestDetails    =  getMaybeManifestDetails(repository.name, manifest)
-      repositoryType          <- identifyRepository(repository, maybeManifestDetails.flatMap(_.repositoryType))
-      maybeDigitalServiceName =  maybeManifestDetails.flatMap(_.digitalServiceName)
-      owningTeams             =  maybeManifestDetails.map(_.owningTeams).getOrElse(Nil)
+      optManifest     <- githubConnector.getFileContent(repository.name, "repository.yaml")
```

Comment:
is it intentional to leave out the counter?

Reply:
Yes, the metrics were for the `github.open` api usage.